dali/operators/image/convolution/laplacian.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_IMAGE_CONVOLUTION_LAPLACIAN_H_
+#define DALI_OPERATORS_IMAGE_CONVOLUTION_LAPLACIAN_H_
+
+#include <memory>
+#include <vector>
+
+#include "dali/pipeline/operator/operator.h"
+#include "dali/pipeline/util/operator_impl_utils.h"
+#include "dali/operators/image/convolution/laplacian_params.h"
+#include "dali/operators/image/convolution/convolution_utils.h"
+#include "dali/pipeline/operator/common.h"
+
+namespace dali {
+
+#define LAPLACIAN_CPU_SUPPORTED_TYPES \
+  (uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, uint64_t, int64_t, float16, float, double)
```

Comment:
Mhh, I don't think we have many operators supporting `double`. I'm not sure if we need to support it here.

Reply:
Sure, let me remove it (esp. as intermediate type is float32 anyway)

Reply:
done